Cubical/Categories/Category/Base.agda

```diff
@@ -12,7 +12,7 @@ private
 
 -- Categories with hom-sets
 record Category ℓ ℓ' : Type (ℓ-suc (ℓ-max ℓ ℓ')) where
-  -- no-eta-equality ; NOTE: need eta equality for `opop`
+  no-eta-equality -- ; NOTE: need eta equality for `opop`
```

Comment:
Why put this back in? It's nice to have `C = C^op^op` definitionally...

Reply:
Oh this was an accident. I rebased from a branch that I use in a project of my own, where I disabled eta-equality for categories [for this reason](https://github.com/agda/agda/issues/4573#issuecomment-1113319069).